nosql.go

```diff
@@ -3,6 +3,7 @@ package nosql
 import (
 	"github.com/pkg/errors"
 	"github.com/smallstep/nosql/badger"
+	badgerV2 "github.com/smallstep/nosql/badgerV2"
```

Comment:
Do those two import compile in all supported architecture? If they do not, we need to split this file by architecture, see https://golang.org/pkg/go/build/#hdr-Build_Constraints

Reply:
I made darwin, linux, arm64, and armv7 binaries of `certificates` using this code. Looks like all our supported architectures are ok. Anything else I should try?

Reply:
No it should be ok.